omigami/utilities/spectrum_df_helper.py

```diff
@@ -0,0 +1,67 @@
+from typing import List, Optional, Tuple
+
+import pandas as pd
+from matchms import Spectrum
+from pandas.tests.groupby.test_value_counts import df
+
+
+class SpectrumDataFrameHelper:
+    def __init__(self):
+        pass
+
+    @staticmethod
+    def scale(spectra_df: pd.DataFrame, min=0, max=1, num_peaks=5000) -> pd.DataFrame:
```

Comment:
do we need to make min/max parameters? Seems like we won't change this.

Reply:
This is user facing, so who knows?

Reply:
`matchms` normalizes (i.e. scales) the peaks from 0 to 1 and it doesn't expose this parameter AFAIK. we do that to all our processed spectra

Reply:
I think we can leave this as it won't hurt to have it. Joe (or anyone else) could, for instance, want to scale from 0 to 20 to have a better scaled plot. I will change the order of arguments os num_peaks, which is more important, comes first, though.

Reply:
ok. I would not expose it unless we need it based on the recent topics I brought on the weekly review call. I think we will remove them in the future because of being unused. But let's leave them if you think they might be used.

It's missing type hints btw